cypress/integration/seededFlows/adminFlows/config/authenticationSection.spec.js

```diff
@@ -127,5 +115,28 @@ describe('Authentication Section', () => {
         cy.findByLabelText('Facebook enabled').should('be.visible');
       });
     });
+
+    it('generates error message when update fails', () => {
+      cy.intercept('POST', 'admin/settings/authentications', {
+        error: 'some error msg',
+      });
+      cy.get('@user').then(() => {
+        cy.visit('/admin/customization/config');
+        cy.findByTestId('authSectionForm').as('authSectionForm');
+
+        cy.get('@authSectionForm').findByText('Authentication').click();
+        cy.get('#facebook-auth-btn').click();
+        cy.get('#settings_authentication_facebook_key').type('randomkey');
+        cy.get('#settings_authentication_facebook_secret').type('randomsecret');
+
+        cy.get('@authSectionForm').findByText('Update Settings').click();
```

Comment:
just a suggestion and definitely not blocking, but it's a bit more robust to use `cy.findByRole` or `cy.findByLabelText` wherever possible, rather than getting by ID selector or inner text

Reply:
Yes @aitchiss Nick also pointed this out to me yesterday 😊 

These cypress tests were existing before my modifications, and the use of `cy.get(#id)` is quite plentiful throughout several files. We decided that making all these tests more robust was beyond the scope of this PR.